src/Sdk/DTPipelines/Pipelines/ContextData/DictionaryContextData.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using System;
+using System.Runtime.Serialization;
```

Comment:
nit: revert this

Reply:
+1 since this was copied from the service side code.